### PR TITLE
PMM-9125 stream parse config

### PR DESCRIPTION
**Pull request by BupycHuk**

### CODE REVIEW

**Review by percona-csalguero — APPROVED** [review]

File: http_client.go [hunk 1]
```diff
@@ -39,7 +39,7 @@ type HTTPClientConfig struct {
 	// FollowRedirects specifies whether the client should follow HTTP 3xx redirects.
 	// The omitempty flag is not set, because it would be hidden from the
 	// marshalled configuration when set to false.
-	FollowRedirects bool `yaml:"follow_redirects"`
+	FollowRedirects bool `yaml:"follow_redirects,omitempty"`
```
Comment (artemgavrilov) [inline http_client.go]: In original Prometheus config they don't use `omitempry` for that field: https://github.com/prometheus/common/blob/ce7006ede1b27960d267508d7ae0af2f51422610/config/http_config.go#L197

Also what it should mean if this field Is omitted?

Comment (artemgavrilov) [reply http_client.go]: Also this field has a comment which mentions `omitmepty`

Comment (BupycHuk) [reply http_client.go]: that's weird. It makes sense if it's pointer, but it's not.

Comment (artemgavrilov) [reply http_client.go]: Well, I propose to stay as close to the original Prometheus config, as possible.

**Comment by artemgavrilov** [comment]
@BupycHuk Ping

**Pull request merged**